api/v1/queries.go

```diff
@@ -0,0 +1,318 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/iancoleman/strcase"
+)
+
+func makeStatusQuery(chainID string) string {
```

Comment:
why the lint here, is this one meant to use a different chain ID than the one provided?

Reply:
nvm resolved in a later commit

Reply:
Hrm. I think I removed that in a future commit. But I tried to keep "oasis-3" (and equivalent) as the chain ID when in places that are surfaced to the user, but it needs "oasis_3" (and equivalent) to be a schema name since "-" isn't a valid character for that.